src/utils/xhr.js

```diff
@@ -320,21 +325,9 @@ export class FetchResponse {
    * Drains the response and returns the JSON object.
    * @return {!Promise<!JsonObject>}
    */
-  json() {
-    return /** @type {!Promise<!JsonObject>} */ (
-      this.drainText_().then(parseJson)
-    );
-  }
-
-  /**
-   * Drains the response and returns a promise that resolves with the response
-   * ArrayBuffer.
-   * @return {!Promise<!ArrayBuffer>}
-   */
-  arrayBuffer() {
```

Comment:
Unused?

Reply:
Yes... also it seems to return a Uint8Array instead of an ArrayBuffer

Reply:
I'll rollback the changes to XHR for this PR. I'm seeing some opportunities to remove polyfills